README.md

```diff
@@ -46,11 +46,17 @@ You can define custom routes that combine to a single file. This allows you to p
 * `*` supports a single path segment in the URL. In the configuration below, `/baz.html` would match but `/bar/baz.html` would not.
 * `**` supports any length in the URL.  In the configuration below, both `/route/foo` would work and `/route/foo/bar/baz`.
 
+There's also an `excepts` option you can use in conjuction with `**`, to match everything except routes in `excepts`.
+
 ```json
 {
   "routes": {
     "/*.html": "index.html",
     "/route/**": "bar/baz.html"
+    "/spa/**": {
+      "path": "index.html",
+      "excepts": ["/spa/assets/"]
```

Comment:
I feel like this exposes a lot of information about which order we (well, `nginx`) resolve the various config options in. I'm tempted to say that we should move all of the request config handling to `ngx_mruby`, and implement this using routes as the last stop, instead, so that exceptions to routes for files, proxy patterns, &c happen "naturally".

We should still document the precedence, but _as a user_ I don't want to have to think about how to control the precedence like this.